src/components/Question/Question.css

```diff
@@ -67,10 +67,37 @@
 }
 
 .buttons-box{
-  grid-area: 6/ 4/ 7 / 8;
+  grid-area: 5/ 4/ 7 / 8;
   display: inline-flex;
   justify-content: space-between;
   margin: 4%;
+  position: relative;
+}
+
+.bx{
```

Comment:
this code is what solves the issue when the back button doesn't display on the DOM.